plugins/modules/monitoring/airbrake_deployment.py

```diff
@@ -48,7 +48,12 @@
     type: str
   revision:
     description:
-      - A hash, number, tag, or other identifier showing what revision was deployed
+      - A hash, number, tag, or other identifier showing what revision from version control was deployed
+    required: false
+    type: str
+  version:
+    description:
+      - A string identifying what version was deployed
```

Comment:
```suggestion
      - A string identifying what version was deployed
    version_added: 1.0.0
```

Reply:
I added `version_added`, but put it at the end of the `version` block, and quoted, to match the other `version_added` fields earlier in this file.

Thanks!

Reply:
Perfect :) With the suggestion feature it was easier to place the suggestion here, the place you put it is even better.